Azure/sonic-buildimage
# [sonic-sairedis]: Update submodule on 201709 branch

**- What I did**
Update sonic-sairedis for Nephos.

**- How I did it**
Update src/sonic-sairedis submodule.

**- How to verify it**
Run the following commands:
make configure PLATFORM=nephos
make target/sonic-nephos.bin
Then install image to check system and network feature is worked as well

**- Description for the changelog**
Update submodule on 201709 branch for Nephos

**- A picture of a cute animal (not mandatory but encouraged)**

Signed-off-by: Wade He <chihen.he@gmail.com>
